### ISSUE TITLE
Provide overwrite control on bundle level [PAXRUNNER-86]

### ISSUE THREAD

**Issue by ops4j-issues** [ops4j/org.ops4j.pax.runner#108]
**[Alin Dreghiciu](https://ops4j1.jira.com/secure/ViewProfile.jspa?accountId=557058:a786db11-2013-41fb-af1d-ca5eb8f74aa0)** created **[PAXRUNNER-86](https://ops4j1.jira.com/browse/PAXRUNNER-86)** 

Currently the --overwrite options do not provide finer control on which bundles to be overwritten in cache. This is quite annoying during development as in that case as a developer one would only need to overwrite the developed bundles not all "user" bundles as for example other bundles that you are deploying together with your under development bundles, especially when those bundles are not in your local repository so are downloaded from the net.

To avoid this it will be good to have finer control by extending the options on bundle scanner level to accept an `@update` option and on the bundles installed via scan-file.

---

**Fixed in:** 0.5.6
Votes: 0, Watches: 0

**Comment by ops4j-issues** [comment]
**[Alin Dreghiciu](https://ops4j1.jira.com/secure/ViewProfile.jspa?accountId=557058:a786db11-2013-41fb-af1d-ca5eb8f74aa0)** commented

All the scanners supports now an "`@update`" (as start/start level) option that will make the provisioning service to force an update on the installed bundles. Using this functionality, Runner via the decorated Bundle, will use the update call to mark the bundle as to be overwritten, so implementing the wanted behavior.



So, bottom line if `@update` specified on one of the scan-file, scan-dir, scan-pom urls the scanned bundles will be overwritten. Same goes for the bundle url's specified in a file scanned by scan-file.<br /> Documentation will be updated to reflect this change.
